config.toml

```diff
@@ -263,3 +263,36 @@ weight = 5
 
 [languages.ru.params]
 language_alternatives = ["en"]
+
+# French
+[languages.fr]
+title = "Helm"
+description = "Helm - Le manageur de paquets de Kubernetes."
+contentDir = "content/fr"
+languageName = "French"
```

Comment:
I would suggest making this line:

` languageName = "Français French"`

You can look at the drop-down on https://deploy-preview-1051--helm-merge.netlify.app/ to see how this will appear. 
![image](https://user-images.githubusercontent.com/2104453/112696499-ecba6380-8e53-11eb-8e27-cd9ed6335566.png)

Reply:
Yeah I was wondering what was the best solution. I thought there was this double syntax because the name of the language was written with non latin letters.